MetaMask/metamask-extension
# Adding border radius full to `Box` component

## Explanation

Currently, there is no border radius prop option for a "full" pill like border radius.

This is a problem because currently engineers have to add extra css to achieve this and there are many instances where this would be useful

In order to solve this problem, this pull request adds a `full`(`9999px`) border radius option. It also separates the options from the main `SIZE` const in the design-system constants. It also updates tests, docs and stories.

## More Information

* Fixes #16116 
Some naming conventions from other libraries

## Screenshots/Screencaps

### After

<img width="1439" alt="Screen Shot 2022-10-06 at 6 11 50 PM" src="https://user-images.githubusercontent.com/8112138/194445710-4d9741a1-77ab-4b2b-8db6-282f4da9f31e.png">
<img width="1440" alt="Screen Shot 2022-10-06 at 6 12 43 PM" src="https://user-images.githubusercontent.com/8112138/194445715-7511cdef-9b51-44a5-b479-20b5d5c7d34e.png">


## Manual Testing Steps

- Go to the latest build of storybook on this PR
- Search `Box` component in search bar
- Check storybook controls for borderRadius work
- Check documentation for borderRadius in the Docs tab

## Pre-Merge Checklist


### + If there are functional changes:



### I will update all the current uses of the SIZE const for borderRadius and deprecate SIZES.NONE in another PR

![Screen Shot 2022-10-06 at 4 58 35 PM](https://user-images.githubusercontent.com/8112138/194439327-53c89305-5c29-4399-a9a8-1c5b31965f5d.png)
![Screen Shot 2022-10-06 at 5 00 42 PM](https://user-images.githubusercontent.com/8112138/194439329-9ea8b395-62f8-4637-beeb-04c03647e43f.png)
